### ISSUE TITLE
Element input with mouse pointer locked

### ISSUE THREAD

**Issue by leonidaspir** [playcanvas/engine#3718]
Right now when the mouse pointer is locked you aren't really able to receive input in world placed UI elements (e.g. 3D screens with buttons).

It would be cool to support this one way or another, interface elements placed in world screens are quite a thing in many first person games.

One easy way for developers to approach this, is by using an internal/private method from the pc.ElementInput class. Example code:

```
        this.app.mouse.on("mouseup", function (e) {

            // --- if pointer is locked we handle our own element input
            if(pc.Mouse.isPointerLocked() === true & e.button !== pc.MOUSEBUTTON_LEFT){

                const centerX = this.app.graphicsDevice.canvas.width / 2;
                const centerY = this.app.graphicsDevice.canvas.height / 2;

                const target = this.app.elementInput._getTargetElement(this.entity.camera, centerX, centerY);

                if(target) target.fire('click');
            }
        }, this);  
```
Basically this is a feature request to expose the `_getTargetElement` method as public API, unless there is formal support for this feature to be discussed.

**Comment by Maksims** [comment]
Be able to do "ray > input" picking, would be definitely useful in various cases.
